src/Apps/Order2/Routes/Checkout/Components/PaymentStep/Order2PaymentForm.tsx

```diff
@@ -117,24 +117,32 @@ export const Order2PaymentForm: React.FC<Order2PaymentFormProps> = ({
     },
     appearance: {
       variables: {
+        // https://docs.stripe.com/elements/appearance-api#variables
         accordionItemSpacing: "10px",
         fontFamily: theme.fonts.sans,
-        colorPrimary: theme.colors.mono100, // accordion is selected
-        colorTextSecondary: theme.colors.mono100, // accordion is not selected
+        colorPrimary: theme.colors.mono100, // A primary color used throughout the Element. Accordian selected.
+        colorTextSecondary: theme.colors.mono60, // The color used for text of secondary importance. Accordian not selected.
+        colorText: theme.colors.mono60, // The default text color used in the Element. Used for input labels.
```

Comment:
minor - the second half of these comments is helpful but the first part gets in the way a bit. I might also put them on separate lines but that is personal taste